README.md

```diff
@@ -3,6 +3,10 @@
 PR Lint checks for a pivotal story id in the branch name. It uses the pivotal API so any random
 pivotal id will lead to check failure.
 
+It adds pivotal story details to the PR description
+
+![pr-lint](https://assets1.cleartax-cdn.com/cleargst-frontend/misc/1568800226_pr-lint.png)
```

Comment:
[optional] an example with a pivotal story description rather than the oops :P

Reply:
actually it was intentional. :D